netci.groovy

```diff
@@ -66,27 +70,31 @@ class WcfUtilities
     
     // Outerloop jobs for WCF Core require an external server reference
     // This should be run 
-    def addWcfOuterloopTestServiceSync(def job, String os, boolean isPR) { 
+    def addWcfOuterloopTestServiceSync(def job, String os, String branch, boolean isPR) { 
         wcfRepoSyncServiceCount++
 
         def operation = isPR ? "pr" : "branch"
 
         job.with { 
             parameters {
+                stringParam('WcfServiceUrl', "http://wcfcoresrv2.cloudapp.net/WcfService${wcfRepoSyncServiceCount}", 'Wcf OuterLoop Test Service Uri')
                 stringParam('WcfRepoSyncServiceUrl', "http://wcfcoresrv2.cloudapp.net/PRService${wcfRepoSyncServiceCount}/pr.ashx", 'Wcf OuterLoop Test PR Service Uri')
             }
         }
-        if (os.toLowerCase().contains("windows")) {
-            job.with { 
-                steps {
-                    batchFile(".\\src\\System.Private.ServiceModel\\tools\\scripts\\sync-pr.cmd ${operation} %WcfRepoSyncServiceUrl%")
-                }           
-            }
-        } 
-        else {
-            job.with { 
-                steps {
-                   shell("HOME=\$WORKSPACE/tempHome ./src/System.Private.ServiceModel/tools/scripts/sync-pr.sh ${operation} \$WcfRepoSyncServiceUrl")
+        // Exclude rc2 branch, since that branch will not have the sync scripts in
```

Comment:
There's a bunch of considerations here: 

* PRs before xxx could blow up a PRService 
* Syncing to rc2, rc1, etc. could blow up the machine
* ... and there are numerous other scenarios that will blow up the machine and require manual intervention which I don't care to enumerate here for now. 

In short, yes, I think there's a lot to consider for later discussion :smile: